apps/test/unit/code-studio/components/BubbleChoiceTest.js

```diff
@@ -69,19 +69,18 @@ describe('BubbleChoice', () => {
         DEFAULT_PROPS.level.previous_level_url + window.location.search
       );
 
-      const continueButton = wrapper.find('button').at(1);
-      assert.equal('Continue', continueButton.text());
-      continueButton.simulate('click');
+      const finishButton = wrapper.find('button').at(1);
+      assert.equal('Finish', finishButton.text());
+      finishButton.simulate('click');
       expect(utils.navigateToHref).to.have.been.calledWith(
-        DEFAULT_PROPS.level.next_level_url + window.location.search
+        DEFAULT_PROPS.level.redirect_url + window.location.search
       );
     });
 
-    it('redirect to script page if no previous/next levels', () => {
+    it('redirect to script page if no previous level', () => {
```

Comment:
Should we still be testing the redirect url if there is no next level? Otherwise, I think we should mention in the test name that it is only looking at the back button, bc I was initially confused why the finish button would care that there was no previous level.

Reply:
having no `redirect_url` is an undefined state and shouldn't occur in practice, but i guess having the finish button remain and redirect back to the script page is probably a better fallback than having no "forward" button. i'll revert that.